apps/docs/src/pages/components/checkbox.mdx

```diff
@@ -123,79 +79,10 @@ export default function Demo() {
 }
 ```
 
-### External error message
```

Comment:
this is a awkward example to add for each form component. it should be added in a complete form example/template instead

Reply:
Maybe we should add a callout on each form component documentation page to direct people to the `FormElement` page (not just on the related components section)